res/values/strings.xml

```diff
@@ -679,6 +679,12 @@
     <string name="qrscan_contains_url">Scanned QR code URL:\n\n%1$s</string>
     <string name="qrscan_fingerprint_label">Fingerprint</string>
     <string name="qrscan_x_verified_introduce_myself">%1$s verified, introduce myself…</string>
+    <string name="withdraw_verifycontact_explain">This QR code can be scanned by others to setup a contact with you.\n\nIf you do not want that any longer, you can withdraw it. You can revive it by scanning it again.</string>
+    <string name="withdraw_verifygroup_explain">This QR code can be scanned by others to join the group \"%1$s\".\n\nIf you do not want that any longer, you can withdraw it. You can revive it by scanning it again.</string>
+    <string name="withdraw_qr_code">Withdraw QR code</string>
```

Comment:
```suggestion
    <string name="withdraw_qr_code">Yes</string>
```

Reply:
ui-wise, "yes" and "no" buttons are not always the best choice, cmp. https://developer.apple.com/design/human-interface-guidelines/ios/views/alerts/

just saying, i know, we are not following these guidelines everywhere :)

Reply:
They went full circle with that. Curiously the image above that text has fallen out of line with its advice.
I use it for the single option notification.
Some of that advice is good reason for going with my suggestions.
The alternative is leaving it up there and going with GNOME style imperative buttons.
The difference between "Yes" and "OK" isn't the problem. Exacerbating the problem of overloading info is.
It doesn't matter what the buttons say if the user is gone by then.